pkg/client/observatorium/observability_config.go

```diff
@@ -123,6 +124,12 @@ type ObservabilityCloudwatchLoggingConfigCredentials struct {
 	SecretAccessKey string `yaml:"aws_secret_access_key" validate:"required"`
 }
 
+type ObservabilityEnterpriseCloudwatchLoggingConfigCredentials struct {
+	AccessKey       string `yaml:"aws_access_key" validate:"-"`
```

Comment:
Can we reuse `ObservabilityCloudwatchLoggingConfigCredentials`?

Reply:
:+1: